src/common/FileSystem.cpp

```diff
@@ -674,6 +680,7 @@ void File::SetLineBuffered(bool enable, std::error_code& err) const
 		ClearErrorCode(err);
 }
 
+#if !defined(BUILD_VM)
```

Comment:
Is there some difference between `defined(BUILD_ENGINE)` and `!defined(BUILD_VM)` ?

Reply:
Ah no, I first wrote it the `!defined(BUILD_VM)` way then rewrote as `defined(BUILD_ENGINE)` but forgot this occurrence, I will fix that.

Reply:
Fixed.